lib/place_type_invalid.rb

```diff
@@ -0,0 +1,11 @@
+class PlaceTypeInvalid < StandardError
+    attr_accessor :message
+
+    def initialize(message)
+        @message = message
+    end
+
+    def to_s
+        "[#{message}] #{super}"
+    end
```

Comment:
This is fine, but I don't think you needed to do anything beyond the class declaration and superclass. What's inherited from StandardError is often enough; the important part is the ability to `rescue` this specific error, which the class itself provides.